Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md

```diff
@@ -0,0 +1,23 @@
+Displays a geographical location in an indicator field using Google Maps.
+
+You need to [Setup Google Maps](https://xsoar.pan.dev/docs/reference/articles) to work with Cortex XSOAR. You can then add this automation to an indicator layout. When viewing the indicator you can see the location in the indicator field.
+
+## Script Data
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | python |
+| Tags | dynamic script |
```

Comment:
The required tag is `dynamic-indicator-section`.

<img width="811" alt="Screen Shot 2022-01-08 at 10 06 02 PM" src="https://user-images.githubusercontent.com/91506078/148670241-5f26d6e1-d731-40a9-a430-0a7ebc28ea41.png">
.